app/models/service_request.rb

```diff
@@ -84,6 +85,11 @@ def self.filter_by_search_311(params)
     requests
   end
 
+  def public_servant_description_validation
+      if public_servant_id == nil
+        public_servant_description = "No aplica"
```

Comment:
Useless assignment to variable - `public_servant_description`. Did you mean `public_servant_id`?